.serena/memories/ci_workflows.md

```
@@ -0,0 +1,25 @@
+# Raven — CI/CD Workflows
+
+## GitHub Actions Workflows
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)